airflow/models/taskinstance.py

```diff
@@ -376,12 +377,13 @@ def log_filepath(self):
     def log_url(self):
         iso = quote(self.execution_date.isoformat())
         base_url = configuration.conf.get('webserver', 'BASE_URL')
-        return base_url + (
-            "/log?"
-            "execution_date={iso}"
-            "&task_id={task_id}"
-            "&dag_id={dag_id}"
-        ).format(iso=iso, task_id=self.task_id, dag_id=self.dag_id)
+
+        relative_url = "/log?execution_date={iso}&task_id={task_id}&dag_id={dag_id}".format(
+            iso=iso, task_id=self.task_id, dag_id=self.dag_id)
```

Comment:
Can you url-quote these three please?
```suggestion
            iso=quote_plus(iso), task_id=quote_plus(self.task_id), dag_id=quote_plus(self.dag_id))
```

And 
```
from six.moves.urllib.parse import quote_plus
```

at the top